services/api-server/src/simcore_service_api_server/exceptions/backend_errors.py

```diff
@@ -0,0 +1,99 @@
+from fastapi import status
+
+from ._base import ApiServerBaseError
+
+
+class BackEndError(ApiServerBaseError):
+    """Base class for all backend exceptions"""
```

Comment:
`BaseBackendError`? Even better than the comment :-)

Reply:
Fixed